resources/assets/components/CampaignTable/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class Header extends React.Component {
+  render() {
+    return (
+      <thead>
+        <tr className="table__header">
```

Comment:
Totally not necessary for this pull request, but it'd be lovely to make a general-purpose Table component for this that accepted an array of heading names & a row component and then formatted the data into it... something like this maybe:

```js
<Table headings={['Campaign Name', 'Approved', 'Pending', 'Rejected', 'Deleted']} row={Row} data={data} />
```